src/client/bookmarks/bookmarksReducer.js

```diff
@@ -31,7 +32,15 @@ const bookmarks = (state = initialState, action) => {
         pendingBookmarks: state.pendingBookmarks.filter(bookmark => bookmark !== action.meta.id),
       };
     default:
-      return state;
+      try {
+        return {
+          ...state,
+          list: JSON.parse(localStorage.getItem(USER_METADATA_KEY)).bookmarks || state.list,
+        };
+      } catch (error) {
```

Comment:
Redux state shouldn't change without action.
If you want to load some state on startup from localStorage you need to do this:

1. Call `getUpdatedSCUserMetadata` action there: https://github.com/busyorg/busy/blob/27dd2383806eda8daf46748cbbbb26739d08ced4/src/client/Wrapper.js#L134-L145
2. Add new case there: `case types.UPDATE_SC2_USER_METADATA.SUCCESS:` that will restore the list using the payload from the action.
3. Add a similar case to `editorReducer` restored drafts from the payload.

This way one webpage is loaded drafts and bookmarks will be restored from localStorage.

Reply:
Hi @Sekhmet Thanks for your comment. I've actually already optimized that part.

As you know since the code will be refactored anyway, I implemented a way with a minimal code change.

Now all initial loading (bookmarks & drafts) is working. Could you review it again?

Reply:
step 1 seems needed anyway, i'll do it, so wait for the review.

Reply:
@Sekhmet and @bonustrack 

Now everything's working. Users need to reload just once for their initial login. Other than that, there's no problem.

Check it out and feel free to merge and disconnect sc metadata.